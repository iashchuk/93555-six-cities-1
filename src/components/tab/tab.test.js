import React from "react";
import Tab from "./tab.jsx";
import renderer from "react-test-renderer";

const item = {
  city: `Amsterdam`,
  isActive: true
};

describe(`Tab`, () => {
  it(`renders correctly`, () => {
    const component = renderer
      .create(<Tab label={item.city} isActive={item.isActive} />)
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
