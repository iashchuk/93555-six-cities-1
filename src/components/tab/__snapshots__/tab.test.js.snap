// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab renders correctly 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item"
    href="#"
    onClick={[Function]}
  >
    <span>
      Paris
    </span>
  </a>
</li>
`;
